engine/proofview.ml

```diff
@@ -345,22 +345,12 @@ let tclBREAK = Proof.break
 
 exception NoSuchGoals of int
 
-(* This hook returns a string to be appended to the usual message.
-   Primarily used to add a suggestion about the right bullet to use to
-   focus the next goal, if applicable. *)
-let nosuchgoals_hook:(int -> Pp.t) ref = ref (fun n -> mt ())
-let set_nosuchgoals_hook f = nosuchgoals_hook := f
-
-
-
 (* This uses the hook above *)
```

Comment:
This comment is outdated and should be removed.

Reply:
Fixed thanks.